patches/@rrweb__record@2.0.0-alpha.17.patch

```diff
@@ -26,7 +31,20 @@ index 46ec389fefb698243008b39db65470dbdf0a3857..bfebdc6a3887a3ddae46b21d21896092
      untaintedBasePrototype$1[key] = defaultObj.prototype;
      return defaultObj.prototype;
    }
-@@ -246,6 +254,9 @@ function isCSSImportRule(rule2) {
+@@ -65,10 +78,10 @@ function getUntaintedPrototype$1(key) {
+     if (!win) return defaultObj.prototype;
+     const untaintedObject = win[key].prototype;
+     document.body.removeChild(iframeEl);
+-    if (!untaintedObject) return defaultPrototype;
++    if (!untaintedObject) return defaultObj.prototype;
+     return untaintedBasePrototype$1[key] = untaintedObject;
+   } catch {
+-    return defaultPrototype;
++    return defaultObj.prototype;
```

Comment:
we return slightly different things on different lines
we didn't change this but it's nicer to always return the same thing
NB in a site where I could reproduce the problem it was `if (!untaintedObject) return defaultPrototype;` that was returning